sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUdfs.scala

```diff
@@ -81,6 +82,8 @@ private[hive] case class HiveSimpleUdf(funcWrapper: HiveFunctionWrapper, childre
 
   override def nullable: Boolean = true
 
+  override def checkInputDataTypes(): TypeCheckResult = TypeCheckResult.TypeCheckSuccess
```

Comment:
we can not get the required data types for HiveSimpleUdf. i think that's ok because function's method is created by children's types. so if children's types are mismatched with arguments of function's method, method in HiveSimpleUdf cannot be created.